source/aggregation-tutorials/one-to-one-join.txt

```diff
@@ -0,0 +1,187 @@
+.. _node-aggregation-one-to-one:
+
+===============
+One-to-One Join
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Introduction
+------------
+
+In this tutorial, you can learn how to perform a one-to-one join using
+aggregation with the {+driver-long+}. A one-to-one join queries two different
+collections and combines related data based on one specific field in each
+collection. 
+
+This example uses two collections, ``products`` and ``orders``. The
+``products`` collection contains documents detailing a shop's available products. The
+``orders`` collection contains documents describing an individual order for a
+product at that shop. The goal of the aggregation is to list all orders
```

Comment:
I think move the sentence that states the goal up to the start of this paragraph. Then move the "This example" sentence down and break out the two collections into bullets. My rationale for this feedback is that it will make the copy more skimmable.